packages/@azure/servicebus/data-plane/test/stress/stress_fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -75,6 +68,22 @@ async function sendReceiveMessages(): Promise<void> {
   }
 }
 
+async function sendReceiveMessagesPerClient(sender: Sender, receiver: Receiver): Promise<void> {
+  while (!isJobDone) {
+    for (let i = 0; i < numOfClients; i++) {
```

Comment:
We can remove the for loop as we pulled it outside of this method.

Reply:
Oops, yes that was unnecessary.